templates/partials/summary/summary.html

```diff
@@ -3,23 +3,28 @@
 {%- else -%}
   {% from "components/summary/_macro.njk" import onsSummary %}
 
-  {% set summaryGroups = [] %}
+  {% set summary_groups = [] %}
+  {% set summary_types = content.summary.summary_type %}
+  {% set view_submitted_response = content.summary.view_submitted_response %}
 
   {%- for group in content.summary.groups if group.blocks -%}
-    {% do summaryGroups.append
+    {% do summary_groups.append
       (
         {
+          "summaryTitle": group.section_title,
```

Comment:
At the moment the rendering is not correct when there is no group title. On the submitted response, when you don't have a section title, the group title are h2, otherwise it is h3, which is odd. Check with Rich is there is a way around it. Setting to an empty string won't work either.

The padding is also off as it is expecting a title after each group. We might have to make these into sections of groups instead of a flatten list of group.
<img width="696" alt="image" src="https://user-images.githubusercontent.com/35296336/235710621-b8d48648-05b0-41a6-a862-cf7e23476944.png">

Reply:
As discussed this is an issue in our implementation which will be resolved in another PR